transform/mattermost-analytics/models/marts/product/licenses/_product__licenses__models.yml

```diff
@@ -142,4 +142,59 @@ models:
       - name: company_name
         description: The company's name.
       - name: license_name
-        description: The name of the license or plan of the customer.
\ No newline at end of file
+        description: The name of the license or plan of the customer.
+
+  - name: dim_latest_server_customer_info
+    description: |
+      Contains the latest customer info for each server. Data contains any values available either from
+        - License sources (CWS, legacy licenses) or
+        - Cloud sources (Stripe).
+      In case multiple data rows are available on each of this sources, only the latest entry is considered.
+
+    columns:
+      - name: server_id
+        description: The server's unique id.
+        tests:
+          - unique
+          - not_null
+      - name: license_id
+        description: The ID of the license.
+      - name: installation_id
+        description: The ID of the cloud installation.
+      - name: company_name
+        description: The name of the company that this license has been issued for.
+      - name: contact_email
+        description: The email to be used for contacting the license's holder, as it appears in license data.
+        tags: [ 'pii' ]
+      - name: sku
+        description: The SKU for the license or `Unknown` if it's not known (i.e. in legacy licenses).
+      - name: license_expire_at
+        description: The expiration date of the license. Exists only if license data have been found.
+      - name: is_trial_license
+        description: |
+          Whether the license is a trial license or not. Legacy licenses do not contain this information and this flag 
+          is automatically set to `false`. If no license is found, then it's automatically set to false.
+        tests:
+          - not_null
+      - name: source
+        description: |
+          The source of the license data. One of `CWS`, `Legacy`,  `CWS and Legacy`, 'Stripe'.
+        tests:
+          - accepted_values:
+              values: ['CWS', 'Legacy', 'CWS and Legacy', 'Stripe']
+
+      - name: found_matching_license_data
+        description: | 
+          Boolean flag, indicating whether license data have been found for the current license id. 
+          - True if matching data have been found.
+          - False if no license id has been found or if no matching license exists.
+
+      - name: found_matching_stripe_entry
+        description: | 
+          Boolean flag, indicating whether data for the current installation ID are available in Stripe. 
+          - True if matching data have been found.
+          - False if no data have been found or if there's no installation ID.
```

Comment:
"no data have been found or" condition below checks for `installation_id not null" only